web/src/pages/salesforce/index.js

```diff
@@ -0,0 +1,263 @@
+import React from "react";
+import Layout from "../../components/Layout/Layout";
+import { Container, Col, Row, Image } from "react-bootstrap";
+import Title from "../../components/UI/Title/Title";
+import Subtitle from "../../components/UI/Subtitle/Subtitle";
+import BrandButton from "../../components/UI/BrandButton/BrandButton";
+import "../../styles/main.scss"
+import styled from "styled-components";
+import Accordion from 'react-bootstrap/Accordion';
+import "./salesforce.scss";
+
+const SalesforcePage = ({ data }) => {
+    
+    return (
+        <Layout pageTitle="Salesforce | Awesome Inc">
+            
+            {/* Header section */}
+            <section id="header">
+                <Container fluid>
+                    <Row className="header-section">
+                        <Col className="header-text">
+                            <Title>SALESFORCE CAREER ACCELERATOR</Title>
```

Comment:
@Sisay-su Can you go and remove the all-caps? You can use the class `text-uppercase` instead. 
([Why?](https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99))